packages/react-native-windows-init/src/Cli.ts

```diff
@@ -607,5 +639,21 @@ function isProjectUsingYarn(cwd: string): boolean {
     console.error(initWindowsError);
   }
   endTelemetrySession(initWindowsError);
-  setExitProcessWithError(argv.verbose, initWindowsError);
-})();
+  setExitProcessWithError(!!options.verbose, initWindowsError);
+}
+
+export interface WindowsInitOptions {
+  version?: string;
+  namespace?: string;
+  verbose?: boolean;
+  telemetry?: boolean;
+  language?: 'cpp' | 'cs';
+  overwrite?: boolean;
```

Comment:
`overwrite`, and several other options are never undefined. Being either required, or having a default (which yargs will populate).